azure-pipelines.yml

```diff
@@ -855,12 +857,6 @@ jobs:
         deb_system: debian
         distro: bullseye
   steps:
-  - task: DownloadPipelineArtifact@1
-    displayName: 'Download Build Artifacts'
-    inputs:
-      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
-      artifactName: pypi
```

Comment:
Since https://github.com/Azure/azure-cli/pull/9820 switched from installing using `whl` to `requirements.py3.$(uname).txt`, it is needless to download built `whl`s.